ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+# Bugs and Questions
+
+### Checklist
+
+* [ ] This is not a `victory-native` specific issue. (Issues that only appear in `victory-native` should be opened [here](https://github.com/FormidableLabs/victory-native/issues/new))
+
+* [ ] I have read through the [FAQ](https://formidable.com/open-source/victory/docs/faq) and [Guides](https://formidable.com/open-source/victory/guides/) before asking a question
+
+* [ ] I am using the latest version of Victory
+
+* [ ] I've searched open issues to make sure I'm not opening a duplicate issue
+
+
+### The Problem
+
+Describe the issue you're seeing, and what you expect the behavior to be.
+
+### Reproduction
+
+Please try to reproduce the issue you're seeing in a sandbox. You can [fork this sandbox](https://codesandbox.io/s/m3xo745x2x) to get started. If you can't reproduce your issue in a sandbox, please list steps to reproduce your issue, and include exact version numbers for `victory` and `react`.
```

Comment:
Let's add a second-level fallback, of opening a minimal repository with install + error reproduction steps. This is most often what I ask for and end up solving issues with...